src/com/company/Women.java

```diff
@@ -0,0 +1,49 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Women extends Human {
+    public Women(String name, String surName, float height, float weight) {
+        super(false, name, surName, height, weight);
+    }
+
+    @Override
+    public String toString() {
+        return "Women " + super.toString();
+    }
+
+   /* У женщин есть дополнительный метод - "родить человека" (возвращает экземпляр человека)
+    Он имеет следующую реализацию:
+    Создать новый экземпляр женщины или мужчины с вероятностью 0.5 со следующими свойствами:
+- имя (String) - ввести с консоли
+- фамилия (String) - берется у экземпляра с мужским полом
+- рост (float) - копируется у соответствующего экземпляра с таким же полом + 0.1*(рост экземпляра противоположного
+рода минус рост однородного экземпляра)
+            - вес (float) - аналогично росту*/
+
+    protected Human toGiveBirth(Human human) {
```

Comment:
protected Human toGiveBirth(Human human) => protected Human toGiveBirth(Man man)

in this case, it won't be possible to call this method for two women scenario like Woman.toGiveBirth(anotherWoman)

Reply:
totally agree))